docs/api/features/pcoords.rst

```diff
@@ -78,11 +80,96 @@ Parallel coordinates can take a long time to draw since each instance is represe
 
 The "fast" drawing mode vastly improves the performance of the parallel coordinates drawing algorithm by drawing each line segment by class rather than each instance individually. However, this improved performance comes at a cost, as the visualization produced is subtly different; compare the visualizations in fast and standard drawing modes below:
 
-.. image:: images/fast_vs_slow_parallel_coordinates.png
+.. plot::
+    :include-source: False
```

Comment:
In the docs, only an image is shown so I utilized this directive option to turn off the source code

Reply:
Perfect, this nicely demonstrates fast vs standard parallel coordinates and the source doesn't need to be shown to the user since it's just a demonstration!